mama/c_cpp/src/c/middleware.c

```diff
@@ -83,6 +83,9 @@ mamaMiddleware_convertFromString (const char*  str)
     if (strcasecmp (str, "hms") == 0)
         return MAMA_MIDDLEWARE_HMS;
     
+    if (strcasecmp (str, "reflect") == 0)
```

Comment:
Please revert all changes in this file for the pull request branch - not relevant to change.